cypress/objects/checks.js

```diff
@@ -38,6 +38,10 @@ class checks {
     searchTaskName(nameTitle, text) {
         cy.get(nameTitle).eq(0).type(text);  
     }
+
+    searchOrganizationName() {
```

Comment:
This function is not used and also is not working because it is missing nameTitle and text parameters